compiler/damlc/daml-doc/src/DA/Daml/Doc/Extract.hs

```diff
@@ -354,6 +363,7 @@ getTypeDocs ctx@DocCtx{..} (DeclData (L _ (TyClD _ decl)) doc)
             ad_args = map (tyVarText . unLoc) $ hsq_explicit tcdTyVars
             ad_anchor = Just $ typeAnchor dc_modname ad_name
             ad_constrs = mapMaybe (constrDoc ad_name) . dd_cons $ tcdDataDefn
+            ad_instances = Nothing -- filled out later in 'distributeInstanceDocs'
```

Comment:
I wonder if it would make sense to add a phantom type parameter indicating the “phase”, i.e., whetehr we are before or after `distributeInstanceDocs`.

Reply:
I think it's a bit much, but I'll keep it in mind for the future.